src/App.js

```diff
@@ -1,15 +1,71 @@
 import React from "react";
-
+import Search from "./Search.js";
+import Restaurant from "./Restaurant.js"
 import Bookings from "./Bookings";
 import "./App.css";
 
+const Heading = () => {
+  return (
+    <div>
+      <header className="App-header">CYF Hotel</header>
+    </div>
+  )
+}
+
+const Footer = (props) => {
+  let data = props.address
+  return (
+    <div className="footer">
+      <ul>
+        {
+          data.map(line => {
+            return <li>{line}</li>
+          })
+        }
+      </ul>
+    </div>
+  )
+
+
+}
```

Comment:
What do you think of moving the Heading and Footer components in two separate files, Heading.js and Footer.js. It'll be more future proof in case the components get larger and require more work.